### PR TITLE
add functionalities to the application

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
# Operations
## In this section I have done the followings:
- Added math operations to the calculator.

### CODE REVIEW

File: src/Components/UI/Calculator.js [hunk 1]
```diff
@@ -1,47 +1,110 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prefer-stateless-function, react/destructuring-assignment, no-nested-ternary, max-len */
+// /* eslint-disable */
```
Comment (Munsa1) [inline src/Components/UI/Calculator.js]: - [ ] Kindly remove the disable linters as linters help us write clean industry acceptable code.

File: src/Components/UI/Calculator.js [hunk 2]
```diff
@@ -1,47 +1,110 @@
...
+    } else if (this.state.operation !== null && this.state.total !== null) {
+      if (this.state.next === null) {
+        this.setState({ next: clickedNumber.toString() });
+      } else {
+        // eslint-disable-next-line no-unused-expressions
+        // eslint-disable-next-line react/no-access-state-in-setstate
+        const temp = this.state.next.toString() + clickedNumber.toString();
+        this.setState({ next: temp });
+      }
+    }
```
Comment (Munsa1) [inline src/Components/UI/Calculator.js]: - [ ] Kindly remove the disable linters as linters help us write clean industry acceptable code.

File: src/Components/UI/Calculator.js [hunk 3]
```diff
@@ -1,47 +1,110 @@
...
 import './Calculator.css';
 
 class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+      next: 0,
+      operation: null,
+    };
```
Comment (Munsa1) [inline src/Components/UI/Calculator.js]: - [ ] Upon loading the app for the first time, the calculator does not work. However when I click on th`AC` button it starts to work as expected, kindly fix this glitch in the code.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Mhamad-Raad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Mhamad-Raad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive `PR` description and `Commit` history ✔️  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**